core/src/main/scala/org/broadinstitute/dsde/rawls/webservice/NotificationsApiService.scala

```diff
@@ -3,8 +3,9 @@ package org.broadinstitute.dsde.rawls.webservice
 import akka.http.scaladsl.marshallers.sprayjson.SprayJsonSupport._
 import akka.http.scaladsl.server
 import akka.http.scaladsl.server.Directives._
-import org.broadinstitute.dsde.rawls.model.Notifications.{NotificationType, WorkspaceNotificationType}
-import org.broadinstitute.dsde.rawls.model.{Notifications, WorkspaceName}
+import org.broadinstitute.dsde.rawls.model.WorkspaceName
+import org.broadinstitute.dsde.workbench.model.Notifications
+import org.broadinstitute.dsde.workbench.model.Notifications.{NotificationType, WorkspaceNotificationType, WorkspaceName => NotificationWorkspaceName}
```

Comment:
another potential followup: harmonize the rawls and workbench-libs `WorkspaceName` case classes